### ISSUE TITLE
Design: Come up with a creative icon related to photos rather than using React icon

### ISSUE THREAD

**Issue by batunpc** [humphd/my-photohub#14]
This issue is related to https://github.com/humphd/my-photohub/pull/10#discussion_r1022874166 from the PR raised at https://github.com/humphd/my-photohub/pull/10 

While implementing the PWA UI I have used one of the Apple emojis 📸 in the Navbar
<img width="383" alt="CleanShot 2022-11-16 at 14 53 21@2x" src="https://user-images.githubusercontent.com/71259399/202280786-f16f8584-3c75-4b5a-b8c3-0e8e34c28691.png">

Therefore, maybe something related to photos/photography icon could be a good choice.

Alternatively, somebody could design a logo with various tools out there (Adobe products, Canva etc.) easily available.

**Comment by mnosov622** [comment]
Hi !

I could come up with the logo. Just let me know the preferences @humphd :)

**Comment by mnosov622** [comment]
I've an idea for a logo:

![image](https://user-images.githubusercontent.com/73063742/202959914-2a56f953-a932-4b9a-b8a2-ba07688a0134.png)

What do you think ?

**Comment by SerpentBytes** [comment]
@humphd @LostButton @batunpc @saminarp @rudychung @anshul137 @devils2ndself, thoughts?

**Comment by LostButton** [comment]
That could work, though personally not a bige fan of that color palette

**Comment by rudychung** [comment]
Looks good, but agree with @LostButton, could maybe switch up the colors.

**Comment by SerpentBytes** [comment]
@mnosov622 First of all, thanks for sharing this with us!

I agree with @LostButton and @rudychung.  We could definitely improve the overall design.  Maybe choose colours that the rest of us can agree on. 

Also, perhaps we could include the project name in the logo? 

Let me know your thoughts!

**Comment by saminarp** [comment]
Thank you @mnosov622  for coming up with this! 😃

I also think we can change the colors, experiment with different color palettes and show us different options. Usually, photo-related applications have colorful logos. We can also keep it simple drawing inspiration from other social media logos such as Instagram, Apple, Twitter :)

**Comment by humphd** [comment]
I really like it!  I agree with others that iterating on the colours would be a good idea.  Thanks for leading this effort.

**Comment by mnosov622** [comment]
What colors do you think we should consider for a logo ?

**Comment by humphd** [comment]
No idea, do some research https://99designs.ca/blog/tips/logo-color-meanings/

**Comment by mnosov622** [comment]
![image](https://user-images.githubusercontent.com/73063742/203080671-3abc2b2f-8839-4dd0-8e03-41bec273d745.png)

**Comment by mnosov622** [comment]
Or, perhaps, something lighter ?
![image](https://user-images.githubusercontent.com/73063742/203104730-9496893f-8752-45bf-8902-d46340af1b38.png)

**Comment by mnosov622** [comment]
![image](https://user-images.githubusercontent.com/73063742/203105669-8b567c22-86fb-40e5-a89a-f9e1955141cf.png)

**Comment by SerpentBytes** [comment]
@mnosov622 Just an idea: What if we keep the inner lens (the neon bit) and maybe consider it our log? Could we give each segment a different colour, based on its meaning?

We should rule out the colour combinations which would make the logo hard to see when added to the `navbar`.  
 In the final version, we could change the text's font family/size/weight/position, etc., and restyle the lens if needed.

You may want to look into https://coolors.co/ as it automatically generates a colour palette.

Let me know your thoughts!

**Comment by mnosov622** [comment]
Hi @SerpentBytes .

Got it ! What colors would we prefer ?

**Comment by LostButton** [comment]
> Hi @SerpentBytes .
> 
> Got it ! What colors would we prefer ?

Something that would pop out due to the dark colour of our navbar, maybe a light color pallette

**Comment by mnosov622** [comment]
Let me know if I understand the preferences correctly ?
![image](https://user-images.githubusercontent.com/73063742/203895089-153eadef-477c-4ebe-bd34-a91981ac4d36.png)

**Comment by SerpentBytes** [comment]
@mnosov622. I am not sure. My previous comment was more of a suggestion than a preference...

**Comment by mnosov622** [comment]
Sure !

I like this one. Very soft and quite appealing. What do you say ?
![image](https://user-images.githubusercontent.com/73063742/204049084-22e81ce6-4a24-4d12-a9a1-216c0d29a12b.png)

**Comment by mnosov622** [comment]
Hi guys !

What do you think ?

**Comment by rudychung** [comment]
Not sure if anyone here has played Portal before, but it looks very close to the Aperture Laboratories one. :laughing:
![image](https://user-images.githubusercontent.com/61764259/204311844-0aeb8d21-27cd-4d48-adc0-5c2b4ccebd9a.png)
But it should, given that they're both based on the aperture of a camera. Other than that, I think it looks distinct enough to be recognizable.

**Comment by humphd** [comment]
Agree, I think the previous one seemed more unique.  I wouldn't copy anything existing.

**Comment by mnosov622** [comment]
Oh, I didn't know there's similar logo. What do we do then ?

**Comment by mnosov622** [comment]
Hi guys !

So, what do we do here ?

**Comment by humphd** [comment]
This is a great question, and comes up all the time in open source.  Someone has to decide, and often people put it off, or hope someone else will do it.

I'd say we use the original design, not copy an existing one.

I'd like to hear some updates on the research you did about logo colour choices.  You never responded to that above.

**Comment by mnosov622** [comment]
I didn't copy the existing one. I just took the suggestion above and created a logo with the lenses.

Abou the colors: I think blue color pallete is a good idea for this project to evoke calmness and trust, so that users will use our platform for their needs.

**Comment by humphd** [comment]
> I didn't copy the existing one. I just took the suggestion above and created a logo with the lenses.

Apologies if it seemed like I said you were copying; that's not what I meant.  My point was that we should try to do something unique, now that we know that this other design is already taken.

> Abou the colors: I think blue color pallete is a good idea for this project to evoke calmness and trust, so that users will use our platform for their needs.

Sounds good.  Can you move toward a PR that includes the logo's raw file(s) as well as generating all the static versions we'll need for various platforms/sizes?  You can use something like https://app-manifest.firebaseapp.com/.

**Comment by mnosov622** [comment]
I'm sorry. I understood you the wrong way :)

Sure, I will do that !

**Comment by mnosov622** [comment]
Hi guys !

What do you think of this logo:
![image](https://user-images.githubusercontent.com/73063742/205458835-dc27c641-35ce-42b7-a535-96e0d9952112.png)

**Comment by SerpentBytes** [comment]
@mnosov622 That's creative, and I am digging the choice of colours. I think we should use the project name which is "My Photohub"

**Comment by SerpentBytes** [comment]
@humphd @rudychung @LostButton @saminarp @batunpc

**Comment by SerpentBytes** [comment]
@mnosov622, could you add the logo to the PR you created? Thank you.

**Comment by mnosov622** [comment]
Sure !

**Comment by mnosov622** [comment]
Or experimenting with font:

![image](https://user-images.githubusercontent.com/73063742/205467969-efc7f832-d134-4fda-8f27-c7c64e9c7155.png)

**Comment by SerpentBytes** [comment]
The previous font you used was a little easier on the eyes.

**Comment by SerpentBytes** [comment]
Text using this font could be a little challenging to read for some.

**Comment by mnosov622** [comment]
Yeah agree. What about something like that:

![image](https://user-images.githubusercontent.com/73063742/205468377-c1598a82-6645-48fb-90a4-4d4c2832b3c0.png)

**Comment by SerpentBytes** [comment]
Easier to read. I would wait for feedback from others on this. 👍🏼

**Comment by mnosov622** [comment]
Yeah, sure !

**Comment by SerpentBytes** [comment]
> Hi guys !
> 
> What do you think of this logo: ![image](https://user-images.githubusercontent.com/73063742/205458835-dc27c641-35ce-42b7-a535-96e0d9952112.png)

I was just wondering. Why did you go switch from the original font?

**Comment by mnosov622** [comment]
It's okay :)

The bold one looks much more readable !

**Comment by mnosov622** [comment]
For now, we've come up with this logo:
![image](https://user-images.githubusercontent.com/73063742/205469449-425083e5-cbec-4746-b99b-11b9c9370399.png)

**Comment by saminarp** [comment]
> Yeah agree. What about something like that:
> 
> ![image](https://user-images.githubusercontent.com/73063742/205468377-c1598a82-6645-48fb-90a4-4d4c2832b3c0.png)

I like this one with the bold letters :)

**Comment by mnosov622** [comment]
Hi Samina,

Yeah, that looks great !

**Comment by mnosov622** [comment]
> > Yeah agree. What about something like that:
> > ![image](https://user-images.githubusercontent.com/73063742/205468377-c1598a82-6645-48fb-90a4-4d4c2832b3c0.png)
> 
> I like this one with the bold letters :)

Let me know if we go with this one and I will create icons with different sizes :)

**Comment by SerpentBytes** [comment]
LGTM. ✅ I would wait to hear back from others, though.

**Comment by batunpc** [comment]
> > > Yeah agree. What about something like that:
> > > ![image](https://user-images.githubusercontent.com/73063742/205468377-c1598a82-6645-48fb-90a4-4d4c2832b3c0.png)
> > 
> > 
> > I like this one with the bold letters :)
> 
> Let me know if we go with this one and I will create icons with different sizes :)

I like this one as well 🤩

**Comment by rudychung** [comment]
Looks good!
